components/core/FilePreviewBubble.js

```diff
@@ -0,0 +1,64 @@
+import * as React from "react";
+import * as Constants from "~/common/constants";
+
+import { css, keyframes } from "@emotion/react";
+import { useState } from "react";
+import SlateMediaObjectPreview from "~/components/core/SlateMediaObjectPreview";
+import SlateMediaObject from "~/components/core/SlateMediaObject";
+
+const fadein = keyframes`
+  0%{
+    opacity: 0;
+  }
+  100%{
+    opacity: 1;
+  }
+`;
+
+const STYLES_FILE_PREVIEW_BUBBLE = css`
+  z-index: ${Constants.zindex.tooltip};
+  padding: 8px;
+  border-radius: 4px;
+  background-color: ${Constants.system.black};
+  animation: ${fadein} 200ms ease-out 1;
+  width: 200px;
+  position: absolute;
+`;
+
+const STYLES_FILE_PREVIEW = css`
+  width: 100%;
```

Comment:
`display: block`

Reply:
what's the consequence for not writing `display: block` ?

Reply:
block is width naturally 100%, height 100% doesn't do anything, and then the other two rules are not important anymore.